component/prometheus/exporter/blackbox/blackbox.go

```diff
@@ -81,10 +82,10 @@ func (t TargetBlock) Convert() []blackbox_exporter.BlackboxTarget {
 }
 
 type Arguments struct {
-	ConfigFile         string        `river:"config_file,attr,optional"`
-	Config             string        `river:"config,attr,optional"`
-	Targets            TargetBlock   `river:"target,block"`
-	ProbeTimeoutOffset time.Duration `river:"probe_timeout_offset,attr,optional"`
+	ConfigFile         string            `river:"config_file,attr,optional"`
+	Config             rivertypes.Secret `river:"config,attr,optional"`
```

Comment:
Is the config always sensitive? If not, this should be a rivertypes.OptionalSecret so it can optionally be sensitive depending on the River passed to it.